Style day of week picker

jadebuckwalter opened the pull request:
Style the dropdown menu so it looks nicer and uniform to other dropdowns

jadebuckwalter committed:
Style day of week picker

Style the dropdown menu so it looks nicer and uniform to other dropdowns

psvenk commented on public/js/extraFunctions.js:
```diff
@@ -1167,6 +1167,50 @@ let initialize_tableData_dropdown = function() {
+        bs = "silver";
+      } else if (j === 2 || j === 5) {
+        bs = "black";
+      }
```
What is this variable `bs` for? It doesn't seem to be used anywhere, and anyway the heavy lifting w.r.t. schedule formatting and black/silver is handled elsewhere.

jadebuckwalter committed:
Clean up and comment code

Remove unnecessary code and add/revise comments

jadebuckwalter committed:
Add dropdown functionality

Toggle the appropriate schedule based on the selected day.

jadebuckwalter commented:
Some minor fixes remain:
- [x] If you click out of the dropdown, it should go away
- [x] The arrow should point up if the menu is open
- [x] Change "select day" to the day of the week if it is a weekday

jadebuckwalter committed:
Default to day of week

On weekdays, display the name of the day on the dropdown instead of
"select day"

jadebuckwalter committed:
Toggle dropdown arrow

Point the arrow up when the menu is open, point it down when closed

jadebuckwalter committed:
Work around undefined day of week

Add support for an edge case where the day of week is undefined by
defaulting to "Select Day"

jadebuckwalter committed:
Hide dropdown with outside click

When you click  outside the dropdown when it is open, hide the options

psvenk reviewed (approved):
Looks good to me!
